src/routes/organization/invite/[inviteId]/+page.server.ts

```diff
@@ -0,0 +1,74 @@
+import { checkSessionAuth, deletePath, pushPath, readPath, writePath } from '$lib/server/firebaseUtils.js';
+import { error, redirect } from '@sveltejs/kit';
+import type { PageServerLoad, Actions } from './$types.js';
+import firebaseAdmin from 'firebase-admin';
+import firebaseAdminCredential, { databaseURL } from "$lib/server/firebaseAdminCredential";
+
+if (!firebaseAdmin.apps.length) {
+  firebaseAdmin.initializeApp({
+    credential: firebaseAdmin.credential.cert(firebaseAdminCredential),
+    databaseURL
+  });
+}
+
+export const load = (async ({ cookies, params: { inviteId }, url }) => {
+  // If user is not logged in, redirect to login
+  await checkSessionAuth(cookies, {
+    loginRedirect: url.pathname,
+  });
+
+  const invite = await readPath<Database.Invite>(`/invites/organization/${inviteId}`);
+  if (!invite) throw error(404);
+
+  const publicOrgDetails = await readPath<Database.Organization.Public>(`/organizations/${invite.orgId}/public`);
+  if (!publicOrgDetails) throw error(404);
+
+  return { organizationName: publicOrgDetails.name };
+}) satisfies PageServerLoad;
+
+export const actions: Actions = {
+  accept: async (event) => {
+    const { params: { inviteId }, request } = event;
+    const invite = await readPath(`/invites/organization/${inviteId}`);
```

Comment:
I think we should parameterize `readPath` whenever possible. We've already checked for null once here, but it's technically possible (like if they had two tabs open and already did it) and we should handle those errors explicitly to make debugging easier. 

```suggestion
    const invite = await readPath<Database.Invite>(`/invites/organization/${inviteId}`);
```